pkg/agent/config/config.go

```diff
@@ -63,7 +63,7 @@ func Request(path string, info *clientaccess.Info, requester HTTPRequester) ([]b
 	return requester(u.String(), clientaccess.GetHTTPClient(info.CACerts), username, password)
 }
 
-func getNodeNamedCrt(nodeName, nodePasswordFile string) HTTPRequester {
+func getNodeNamedCrt(nodeName, nodeIP, nodePasswordFile string) HTTPRequester {
```

Comment:
should this really be multiple IPs?

Reply:
I found that the generated certs are generated with all ip's therein. Seemed reasonable at the outset. Happy to adjust though.